prep-javascript-functions/main.js

```diff
@@ -0,0 +1,53 @@
+function addTwoNumbers(num1, num2) {
+  return console.log(num1 + num2);
+}
+
+addTwoNumbers(2, 2);
+
+function convertHoursToMinutes(hours) {
+  return console.log(hours * 60);
+}
+
+convertHoursToMinutes(2);
+
+function getGreeting(name) {
+  return console.log(`Hello ${name} !`);
+}
+
+getGreeting('World');
+
+function addAndMultiplyBy5(num1, num2) {
+  return console.log((num1 + num2) * 5);
```

Comment:
Careful, we don't want to console log inside of our functions. Instead we want to put the return value in a variable and console log the variable like in the example gif.